yogstation/code/game/objects/items/premadepapers.dm

```diff
@@ -107,12 +107,15 @@
 	written += "<span class=\"paper_field\"></span>"
 	written += new/datum/langtext("<br><br>",/datum/language/common)
 	written += new/datum/langtext("<b>Assisting Officer(s) </b><br><br>",/datum/language/common)
-	written += new/datum/langtext("<table><row><cell><b>Rank</b><cell> <b>Name</b><cell> <b>Position</b><br>",/datum/language/common)
+	written += new/datum/langtext("<table><row><cell><b>Rank</b><cell> <b>Name</b><cell> <b>Position</row></b><br>",/datum/language/common)
 	written += new/datum/langtext("<row><cell><b>",/datum/language/common)
 	written += "<span class=\"paper_field\"></span>"
 	written += new/datum/langtext("</b><cell> ",/datum/language/common)
 	written += new/datum/langtext("<cell> ",/datum/language/common)
 	written += "<span class=\"paper_field\"></span>"
+	written += new/datum/langtext("<cell> ",/datum/language/common)
+	written += "<span class=\"paper_field\"></span>"
```

Comment:
You dont need to change it for this time but in the future prefer single quotes instead of escaping double quotes like this:

``written += "<span class='paper_field'></span>"``